src/main/webapp/js/adminActivityLog.js

```diff
@@ -59,7 +59,7 @@ function submitFormAjax(searchTimeOffset) {
     var formObject = $('#ajaxLoaderDataForm');
     var formData = formObject.serialize();
     var button = $('#button_older');
-    var $logsTable = $('#logsTable');
+    var $logsTable = $('#logsTable > tbody');
```

Comment:
`#logsTable` is the ID of existing log table. The Ajax log rows come from `#logs-table`.